Pull request 1

ArtemBorg2010 opened the pull request.

ArtemBorg2010 committed:
Test new Commit

ArtemBorg2010 committed:
deleted Test new Commit

ArtemBorg2010 committed:
Admin GUI added (add car to the list)
Garage: fill() is now a constructor
A rent system fix
Menu Admin GUI
Admin Controller added

Temik-Karpov commented on src/main/java/ru/artem/Online/Market/controllers/AdminController.java:
```diff
@@ -1,4 +1,45 @@
+        return "menu";
+    }
+
+//    @GetMapping("/adminCarAdded")
```
Просьба удалить комментарии.

ArtemBorg2010 replied:
сделал

Temik-Karpov replied:
Можно закрывать комментарий.

Temik-Karpov commented on src/main/java/ru/artem/Online/Market/controllers/AdminController.java:
```diff
@@ -1,4 +1,45 @@
+    Garage garage;
+
+    @Autowired
+    Car car;
```
Странно, что у контроллера есть какая то отдельная машина. Все машины должны храниться внутри гаража, в этом его предназначение.

Temik-Karpov replied:
Тоже самое и в другом контроллере

Temik-Karpov commented on src/main/java/ru/artem/Online/Market/controllers/AdminController.java:
```diff
@@ -1,4 +1,45 @@
+
+    @PostMapping("/adminAddCarProgress")
+    public String addCar(@ModelAttribute("car") Car newCar, Model model){
+        changedCar=newCar;
```
а зачем это нужно вообще? Может changedCar вообще убрать?

ArtemBorg2010 replied:
Согласен, убрал

Temik-Karpov replied:
Можно закрывать комментарий.

Temik-Karpov commented on src/main/java/ru/artem/Online/Market/models/Garage.java:
```diff
@@ -35,8 +35,9 @@ public void fill() {
-        cars = List.of(car1, car2, car3);
+        cars.add(car1);
+        cars.add(car2);
+        cars.add(car3);
```
Я бы заменил на addAll, будет симпатичнее

ArtemBorg2010 replied:
сделал

Temik-Karpov replied:
Можно закрывать комментарий.

ArtemBorg2010 committed:
Admin GUI: New "remove car" option;
Design fix: Replaced links with buttons;
Updated profile: All buttons are working;
Prices and balance: New cash system (Not completed yet), Added price tags to the table
Removed left comments;
Menu Admin GUI updated;
Garage: a better way to fill the car list;
